README.md

```diff
@@ -1,13 +1,45 @@
-# RdTools
+# About RdTools
 
-Degradation Tools
+RdTools is a set of Python tools for analysis of photovoltaic data.
+In particular, high frequency (hourly or greater) PV production data is evaluated
+over several years to obtain rates of performance degradation over time.
 
-Python tools for photovoltaic degradation analysis.
+## Workflow
 
-## Install using pip
+0. Import and preliminary calculations
+1. Normalize data using a performance metric
+2. Filter data that creates bias
+3. Aggregate data
+4. Analyze aggregated data to estimate the degradation rate
+
+
+<img src="./screenshots/Workflow1.png" width="600" height="331" alt="RdTools Workflow"/>
+
+## Degradation Results
+
+RdTools output shows filtered historical data along with a linear trend line.
+However, the most accurate estimate of degradation over the analysis period in question
+is the Year-on-year (YOY) distribution, which is a normal distribution, the central tendancy
+of which is the most representative degradation rate.  Distribution width provides information
+about the accuracy of the analysis.
+
+<img src="./screenshots/Clearsky_result.png" width="600" height="456" alt="RdTools Result"/>
+
+
+Two methods are available for system performance ratio calculation.  The sensor-based
+approach assumes that site irradiance and temperature sensors are calibrated and in good repair.
+Since this is not always the case, a 'Clear-Sky' workflow is provided that is based on
+modeled temperature and irradiance.  Note that site irradiance data is still required to identify
+clear-sky conditions to be analyzed.  In many cases, the 'Clear-Sky' analysis can identify conditions
+of instrument errors, such as in the above analysis.
```

Comment:
I think "Clear-Sky" should be lower case.

Reply:
OK